sdks/go/pkg/beam/runners/prism/internal/worker/bundle.go

```diff
@@ -132,23 +134,66 @@ func (b *B) ProcessOn(ctx context.Context, wk *W) <-chan struct{} {
 		},
 	}
 
-	// TODO: make batching decisions.
-	dataBuf := bytes.Join(b.InputData, []byte{})
+	// TODO: make sending cap decisions, to reduce processing time overhead.
```

Comment:
Non blocking, FYI. I interpreted this TODO as "set a maximum to the the number of b.Input to iterate through to reduce processing time overhead"

Reply:
English is a terrible language. Rewriting to 

	// TODO: make batching decisions on the maxium to send per elements block, to reduce processing time overhead."cap per send",